secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/DataCoordinatorClient.scala

```diff
@@ -233,28 +238,35 @@ class HttpDataCoordinatorClient[CT,CV](httpClient: HttpClient,
   implicit val reCodec = SimpleHierarchyCodecBuilder[Response](NoTag).branch[Upsert].branch[NonfatalError].build
 
   override def postMutationScript(script: JArray, cookie: CookieSchema): Option[Either[RequestFailure, UpdateSchemaFailure]] = {
+    val resourceScope = new ResourceScope("post mutation script")
+
```

Comment:
It'd be better to move this down below (I'll comment where) and manage it with `using`

Reply:
Should I be using simplearm.v2 here? (I noticed other parts of DC using just simplearm

Reply:
Yeah, non-v2 is all legacy code that could do with updating.

Reply:
👍